redaxo/src/core/lib/util/path_default_provider.php

```diff
@@ -233,6 +233,14 @@ public function pluginData($addon, $plugin, $file)
         return $this->addonData($addon, 'plugins/' . $plugin . '/' . $file);
     }
 
+    /**
+     * Returns the path to the log folder.
+     */
+    public function log(string $file): string
```

Comment:
Müssen wir das als BC break dokumentieren für leute mit eigenem path-provider?

Reply:
Solange man von diesem erbt, passt eigentlich alles (so machen wir es bei Yak).
Aber du meinst, wenn man einen komplett eigenen erstellt?

Reply:
Jep, zumindest gibt es hier keinen contract dass man ableiten muss
https://github.com/redaxo/redaxo/blob/8af59be636f1b2d0d0a155b3633213d4fa964cc7/redaxo/src/core/lib/util/path.php#L17

Reply:
Jupp, stimmt schon. Ich werde mit `method_exists` in `rex_path` arbeiten.

Reply:
falls wir andere nicht zulassen wollen könnten wir den phpdoc anpassen.. (aber trotzdem erstmal den harten bc break einbauen)